2.crud_queries/tasks.md

```diff
@@ -1,38 +1,99 @@
 1. Create a database named `blog`.
 
+```js
+use blog;
+```
 2. Create a collection called 'articles'.
-
+```js
+db.createCollection("articles");
+```
 3. Insert multiple documents(at least 3) into articles. It should have fields
+```js
+db.articles.insert({"title": "HTML", "description": "Hyper Text Markup Language", "author": {"name": "xyz", "age": 21, "email": "email@email"}, "tag": ["webdevelopment", "node", "react"]});
+
+db.articles.insert({"title": "CSS", "description": "Cascading Style Sheet", "author": {"name": "abc", "age": 25, "email": "email@email"}, "tag": ["css", "node", "react"]});
 
+db.articles.insert({"title": "Node", "description": "Node Js is used for the backend", "author": {"name": "ghi", "age": 29, "email": "email@email"}, "tag": ["mongo", "node", "express"]});
+```
 4. Find all the articles using `db.COLLECTION_NAME.find()`
+```js
+db.articles.find();
 
-5. Find a document using _id field.
+//Output is: 
+{ "_id" : ObjectId("5ea8600bba80c3c0ecb11403"), "title" : "HTML", "description" : "Hyper Text Markup Language", "author" : { "name" : "xyz", "age" : 21, "email" : "email@email" }, "tag" : [ "webdevelopment", "node", "react" ] }
 
-6. Find documents using title and author's name field.
+{ "_id" : ObjectId("5ea8600bba80c3c0ecb11404"), "title" : "CSS", "description" : "Cascading Style Sheet", "author" : { "name" : "abc", "age" : 25, "email" : "email@email" }, "tag" : [ "css", "node", "react" ] }
 
+{ "_id" : ObjectId("5ea8600bba80c3c0ecb11405"), "title" : "Node", "description" : "Node Js is used for the backend", "author" : { "name" : "ghi", "age" : 29, "email" : "email@email" }, "tag" : [ "mongo", "node", "express" ] }
+```
+5. Find a document using _id field.
+```js
+db.articles.find({_id: ObjectId("5ea8600bba80c3c0ecb11403")});
+```
+6. Find documents using title and author's name field.
+```js
+db.articles.find({}, {"title": "", "name": ""});
```

Comment:
Since author is a nested document, so to query using author's name you should say
```
db.articles.find({title: "", "author.name": ""});
```

Reply:
Sir I have made the changes.